index.html

```diff
@@ -0,0 +1,162 @@
+        <ul class="Desk_nav">
+          <li class="Desk_list"><a href="./about.html"> About </a></li>
+          <li class="Desk_list"><a href="#work"> Program </a></li>
+          <li class="Desk_list"><a href="#contact-form"> Join </a></li>
+          <li class="Desk_list"><a href="#work"> Sponsor </a></li>
+          <li class="Desk_list"><a href="#contact-form"> News </a></li>
+        </ul>
+      </nav>
+      <div class="humburger" d>
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+      <div class="introContainer">
+        <p>"Hello! Great People"</p>
+        <h1>
+          Timbuktu Technical <br />
+          Institute Calling<br />
+          2023
+        </h1>
+        <p class="para3">
+          Timbuktu Technical Institute is calling for people all over the
+          Beautiful African Continent to apply to the free programs it offers.
+          Register on this site for free.
+        </p>
+        <h2>23.02.2023(Thurs) ~ 27(Mon)</h2>
+        <h3>@ Timbuktu Technical Institute offices across the continent</h3>
+      </div>
+    </section>
+    <section class="main">
+      <h1>Main Programs</h1>
+      <hr>
+      <div class="mainActivities">
+        <div class="activity">
+          <img src="./Images/Lecture.png" alt="Lecture Image" />
+          <h2>Lecture</h2>
+          <p>
+            Participate in our lecture program in our various Timbuktu Technical
+            Institute lecture halls across the continent
+          </p>
+        </div>
+        <div class="activity">
+          <img src="./Images/exhibition.png" alt="Exhibition Image" />
+          <h2>TTI exhibition</h2>
+          <p>
+            Participate in our lecture program in our various Timbuktu Technical
+            Institute lecture halls across the continent
+          </p>
+        </div>
+        <div class="activity">
+          <img src="./Images/forum.jpg" alt="Forum Image" />
+          <h2>TTI Forum</h2>
+          <p>
+            Participate in our lecture program in our various Timbuktu Technical
+            Institute lecture halls across the continent
+          </p>
+        </div>
+        <div class="activity">
+          <img src="./Images/workshop.png" alt="Workshop Image" />
+          <h2>Workshop</h2>
+          <p>
+            Participate in our lecture program in our various Timbuktu Technical
+            Institute lecture halls across the continent
+          </p>
+        </div>
+        <div class="activity">
+          <img src="./Images/networking.webp" alt="networking Image" />
+          <h2>Network</h2>
+          <p>
+            Participate in our lecture program in our various Timbuktu Technical
+            Institute lecture halls across the continent
+          </p>
+        </div>
+        <h3 class="join_program_link">Join TTI Program 2023</h3>
+      </div>
+      <h1 class="seeHeader">See the whole Program</h1>
```

Comment:
- [ ] The see the whole program has different color and the text underline is going full width, basically it is different from the main header, please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/217833198-ffb96b16-8c12-4cd7-bc4f-bec25bd3110e.png)

what you have :point_down: 
![image](https://user-images.githubusercontent.com/97521557/217833333-1ad90856-4c42-40f1-9a11-dcc55f650774.png)

Reply:
Okay, I will ensure that I make that change.
Thank you @PinkMoon25